app/views/includes/foot.jade

```diff
@@ -10,14 +10,11 @@ script(type='text/javascript', src='/lib/underscore/underscore-min.js')
 script(type='text/javascript', src='/lib/bootstrap/js/bootstrap.js')
 
 //AngularJS
-script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular.min.js')
-script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-resource.js')
-script(type='text/javascript', src='https://code.angularjs.org/1.1.5/angular-cookies.js')
+script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular.js')
+script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-resource.js')
+script(type='text/javascript', src='http://code.angularjs.org/1.1.5/angular-cookies.js')
 script(type='text/javascript', src='https://cdn.jsdelivr.net/npm/jwt-decode@2.2.0/build/jwt-decode.min.js')
 script(type='text/javascript' src="https://unpkg.com/axios/dist/axios.min.js")
-link(rel="stylesheet" src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/css/materialize.min.css")
```

Comment:
Any reason why you removed the materialize link. I thought we were to use materialize in the project